src/ansys/fluent/core/solver/flobject.py

```diff
@@ -49,6 +49,7 @@ class _InlineConstants:
     user_creatable = "user-creatable?"
     allowed_values = "allowed-values"
     file_purpose = "file-purpose"
+    file_purpose = "file-purpose"
```

Comment:
these are duplicate statements

Reply:
Yes, I pulled latest changes and this was added during resolving conflicts. I'm cleaning stuff like this now. it's in progress.